README.md

```diff
@@ -7,19 +7,25 @@ Similar to [torchdiffeq](https://github.com/rtqichen/torchdiffeq), algorithms in
   <img width="600" height="450" src="./assets/latent_sde.gif">
 </p>
 
-## Requirements
-- torch>=1.6.0
-- blist
-- numpy>=1.19.1
-- boltons>=20.2.1
-
-These requirements are specified in `setup.py` using `install_requires`.
-
 ## Installation
+#### Command
+
 ```shell script
 pip install git+https://github.com/google-research/torchsde.git
 ```
 
+#### Requirements
+Python 3 and PyTorch >=1.6.0
```

Comment:
I guess if `python_requires='~=3.6'`, we might as well state Python >= 3.6 here.